templates/grafana/index.ts

```diff
@@ -0,0 +1,37 @@
+import {
+  Output,
+  Services
+} from "~templates-utils";
```

Comment:
```suggestion
import { Output, Services } from "~templates-utils";
```

Reply:
I totally agree, I recommend using the [prettier](https://prettier.io/) app.